src/platform.c

```diff
@@ -29,6 +29,18 @@ float get_battery_voltage(void)
 	return battery_bits * ADC_LSB * VOLT_DIV_FACTOR;
 }
 
+/**
```

Comment:
The idea is to remove the file platform.c sometime... where would you put this function? Maybe on mmlib library...

Reply:
Which function?

Reply:
The new one 'get_motors_voltage'

Reply:
I do not know. Maybe we should create a `power` module, which holds the `read_battery_voltage` and the `read_motors_voltage` (?).

Reply:
good, I will take that name into account for the new architecture :D